src/SplitPackages/Program.cs

```diff
@@ -150,6 +157,24 @@ private int Execute()
             }
         }
 
+        private void WriteErrorMessage(List<string> errors)
+        {
+            var header = new [] {
+                "The list of packages in the source folder is inconsistent with the list of packages in the CSV file:"
+            };
+                
+            var message = string.Join($"{Environment.NewLine}    ", header.Concat(errors));
+            
+            if(_ignoreErrors.HasValue())
+            {
+                Logger.LogWarning(message);                
+            }
+            else
+            {
+                Logger.LogError(message);
```

Comment:
Do you know if `ILogger` actually writes to the error stream? Our CI is configured to pop errors up to the summary screen so it's nicer than having to dig through build logs.